client/lib/feed-stream-store/feed-stream.js

```diff
@@ -188,6 +189,13 @@ export default class FeedStream {
 			this.selectedIndex = nextIndex;
 			this.emitChange();
 		}
+
+		// If we are getting close to the end of the loaded stream, or are already at the end,
+		// start fetching new posts
+		if ( nextIndex + 4 > this.postKeys.length || nextIndex === -1 ) {
```

Comment:
Thanks for drawing more attention to this.  I actually think it should be far higher than 4 and that would help the Reader experience.  Ideally a user would only see a loading placeholder onFirstLoad and we would prefetch early enough that they wouldn't see it naturally unless scrolling like a madman!